docker/docker-compose.1804.50.yaml

```diff
@@ -7,7 +7,7 @@ services:
     build:
       args:
         ubuntu_version: "bionic"
-        swift_version: "5.0"
```

Comment:
I think `5.0` is right here and means "latest 5.0.x"

Reply:
yes keeping it on 5.0 is better than specifying 5.0.3 otherwise you need to remember to change it when 5.0.4 comes out etc

Reply:
Ok I'll remove it from all of them, 5.1 already specified 5.1.3 so I've removed, unless there's a particular reason to keep?

Reply:
remove from everywhere